app/src/pages/inside/projectSettingsPageContainer/content/notifications/notifications.jsx

```diff
@@ -160,6 +137,18 @@ export const Notifications = () => {
     );
   };
 
+  useEffect(() => {
+    setHeaderTitleNode(
+      <span className={cx('button')} onClick={addNotificationCase}>
+        <Button disabled={!isAbleToEditNotificationCaseList()}>
```

Comment:
Please remove word `case` from names as now we go away from such terminology.
Prev: notification cases
Now: notifications